src/view/com/composer/photos/SelectPhotoBtn.tsx

```diff
@@ -19,16 +19,17 @@ export function SelectPhotoBtn({gallery}: Props) {
   const pal = usePalette('default')
   const {track} = useAnalytics()
   const {requestPhotoAccessIfNeeded} = usePhotoLibraryPermission()
+  const {isDesktop} = useWebMediaQueries()
 
   const onPressSelectPhotos = useCallback(async () => {
     track('Composer:GalleryOpened')
 
-    if (!isDesktopWeb && !(await requestPhotoAccessIfNeeded())) {
```

Comment:
This is actually an issue of the platform not the device size. We should change it to `isNative`